lib/methods/screenshots.ts

```diff
@@ -277,21 +278,31 @@ export async function getFullPageScreenshotsDataDesktop(
     const screenshot = await takeBase64Screenshot(takeScreenshot);
     screenshotSize = getScreenshotSize(screenshot, devicePixelRatio);
 
+    // The actual screenshot size might be slightly different than the inner height
+    // In that case, use the screenshot size instead of the innerHeight
+    if (i === 0 && screenshotSize.height !== actualInnerHeight) {
+      if (Math.round(screenshotSize.height) === actualInnerHeight) {
+        actualInnerHeight = screenshotSize.height;
+      } else {
```

Comment:
Stupid me, there are some drivers that already take a fullpage screenshot, it was with FireFox, but also with SafariDriver for IE11. That's the reason why it is breaking there. Removing the `else` ould be sufficient. We don't need the extra logs. 

If you can remove that one I can merge it. I also need to fix 1 small thing and when that is merged I can create a new release.

Thanks for your work on this @tomyam1 ! 🙏